### DISCUSSION TITLE
Welcome to Qwen-7B-Chat-TensorRT-LLM Discussions!

### DISCUSSION THREAD

**Discussion by Tlntin** [Tlntin/Qwen-TensorRT-LLM#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CedricHwong** [comment]
build过程中出现.....[TensorRT-LLM][WARNING] Cannot profile configuration 167 (for m=4096, n=5120, k=13696). Skipped.... 大家有相同的问题吗 能够正常编译引擎，但是有很多之前没看到过的警告。

**Reply by Tlntin** [reply]
正常

**Reply by CedricHwong** [reply]
python3 run.py
Loading engine from /root/qwen/trt_engines/fp16/1-gpu/qwen_float16_tp1_rank0.engine
Input: "<|im_start|>system
You are a helpful assistant.<|im_end|>
<|im_start|>user
你好，请问你叫什么？<|im_end|>
<|im_start|>assistant
"
Output: "  ( (, (  ( ( ( (*math, Him and, ( ( -,paralle ( under, altri ( (  AT and C  (  ( North G  ( (, Is   orbaugh i   I    (, But " and  questa,  ( ( - (" plata  ( ( , H B In  ( (    ( M, S,, '  ( ( ( ( I taraf ( ? … analyzing still,   ( ( Alla ( (? slot ( K ( ( S and and ( " ( over (  ( ( need, ( ( ( A, ( ( ( ( ( And , and, and ( ( R, -- (  ( ( ( taraf,  (, N   theseik of ( "  ( (, ( A (, ( A (, " (*scale or ( L ( (, (, ( “ ( ,, A ( Po ( ( ,"

**Reply by CedricHwong** [reply]
救。。。
